portfolio/src/main/webapp/script.js

```diff
@@ -26,7 +26,11 @@ function promptRandomQuote() {
   // Pick a random quote.
     const quote = quotes[Math.floor(Math.random() * quotes.length)];
 
-  // Show message.
-  // TO DO: make message a pop up
-    alert(quote)
+  // Return quote as string to be used elsewhere.
+    return quote;
 }
+
+function popupMessage() {
+    var quoteToDisplay = pickRandomQuote();
+    document.getElementById("quoteDisplayer").innerHTML = quoteToDisplay;
```

Comment:
In looks like `pickRandomQuote()` is returning a string and not a formatted html so adding it to div text body might be more appropriate: `.innerText = quoteToDisplay;`